client/components/app/BookShelfToolbar.vue

```diff
@@ -28,13 +28,19 @@
             <span class="font-mono">{{ numShowing }}</span>
           </div>
           <div class="flex-grow" />
-          <ui-btn color="primary" small :loading="processingSeries" class="flex items-center" @click="markSeriesFinished">
+          <ui-checkbox v-model="settings.collapseSeries" label="Collapse Series" checkbox-bg="bg" check-color="white"
+            small class="mr-2" @input="updateCollapseSeries" />
```

Comment:
Updated to use a different setting.

I also added a new commit that moves all of these options to local storage. It took longer than I thought it would, but the utility class I created should help you add any other items you want to move to local storage. Unfortunately, I couldn't get it to work as a mixin, but it isn't too much to add.